pkg/scheduler/core/spreadconstraint/group_clusters.go

```diff
@@ -97,8 +101,10 @@ func groupClustersBasedTopology(
 func groupClustersIngoreTopology(
 	clustersScore framework.ClusterScoreList,
 	rbSpec *workv1alpha2.ResourceBindingSpec,
+	calAvailableReplicasFunc func(clusters []*clusterv1alpha1.Cluster, spec *workv1alpha2.ResourceBindingSpec) []workv1alpha2.TargetCluster,
```

Comment:
the input args calAvailableReplicasFunc is noused

Reply:
yes, it is omissive.